hfuzz-signer/signed-message.rs

```diff
@@ -1,20 +1,5 @@
-use core::convert::TryFrom;
-use filecoin_signer::api::{
-    MessageTx, MessageTxAPI, MessageTxNetwork, SignedMessageAPI, UnsignedMessageAPI,
-};
-use forest_address::Address;
-use forest_crypto::{Signature, Signer};
-use forest_message::{SignedMessage, UnsignedMessage};
-
-struct DummySigner;
-
-impl Signer for DummySigner {
-    fn sign_bytes(&self, _: &[u8], _: &Address) -> Result<Signature, Box<dyn std::error::Error>> {
-        Ok(Signature::new_secp256k1([0u8].to_vec()))
-    }
-}
-
 fn main() {
+    /* FIXME
```

Comment:
what's the issue encountered here?

Reply:
The fuzzer was on a now removed structure. We now use `SignedMessage` from `extras`. I didn't want to include this in the PR as it is already big and I haven't done much fuzzing work.

I will create a issue for re-introduce fuzzing. Also we now used directly structure from the fvm_shared lib so they might be already doing just that.